src/js/structs/__tests__/ServiceTree-test.js

```diff
@@ -147,6 +148,79 @@ describe('ServiceTree', function () {
     });
   });
 
+  describe('#filterItemsByFilter', function () {
+
+    beforeEach(function () {
+      this.instance = new ServiceTree({
+        id: '/group/id',
+        apps: [
+          {
+            id: 'alpha',
+            cmd: 'cmd'
+          },
+          {
+            id: 'beta',
+            cmd: 'cmd',
+            labels: {DCOS_PACKAGE_FRAMEWORK_NAME: 'beta'},
+            tasksHealthy: 1,
+            tasksRunning: 1
+          },
+          {id: 'gamma', cmd: 'cmd', labels: {RANDOM_LABEL: 'random'}}
+        ],
+        groups: [
+          {
+            id: '/test', apps: [
+              {id: 'foo', cmd: 'cmd'},
+              {id: 'bar', cmd: 'cmd'}
+            ], groups: []
+          }
+        ],
```

Comment:
Would be nice to have the indentation agree on one format throughout!  I suggest something like
```
apps: [
	{
	  id: 'alpha',
	  cmd: 'cmd'
	},
	{
	  id: 'beta',
	  cmd: 'cmd',
	  labels: {DCOS_PACKAGE_FRAMEWORK_NAME: 'beta'},
	  tasksHealthy: 1,
	  tasksRunning: 1
	},
	{
	  id: 'gamma', 
	  cmd: 'cmd', 
	  labels: {
	  	RANDOM_LABEL: 'random'
  	  }
  	}
],
groups: [
	{
	  id: '/test', apps: [
	    {
	   	  id: 'foo', 
	   	  cmd: 'cmd'
	   	},
	    {
	      id: 'bar', 
	      cmd: 'cmd'
	    }
	 ], 
	 groups: []
	}
],
```
what do you think?